omim2obo/parsers/omim_txt_parser.py

```diff
@@ -12,22 +13,57 @@
 # from rdflib import URIRef
 
 from omim2obo.config import config, DATA_DIR
+from omim2obo.namespaces import RO
 from omim2obo.omim_type import OmimType
-# from omim2obo.omim_client import OmimClient
-# from omim2obo.omim_code_scraper.omim_code_scraper import get_codes_by_yyyy_mm
 
 
 LOG = logging.getLogger('omim2obo.parser.omim_titles_parser')
-
-
-def retrieve_mim_file(file_name: str, download=False, return_df=False) -> Union[List[str], pd.DataFrame]:
+# MIM_NUMBER_DIGITS = 6  # if you see '6' in a regexp, this is what it refers to
+# MORBIDMAP_PHENOTYPE_MAPPING_KEY_MEANINGS: The Monarch Ingest and Kevin S have identified some ECO properties that
+# ...probably apply to each of these. See:
+# https://github.com/monarch-initiative/monarch-ingest/blob/main/monarch_ingest/ingests/omim/omim-translation.yaml
+# "1": "inference from background scientific knowledge used in manual assertion"
+# "2": "genomic context evidence"
+# "3": "sequencing assay evidence"
+# "4": "sequencing assay evidence"
+MORBIDMAP_PHENOTYPE_MAPPING_KEY_MEANINGS = {
+    '1': 'The disorder is placed on the map based on its association with a gene, but the underlying defect is '
+         'not known.',
+    '2': 'The disorder has been placed on the map by linkage or other statistical method; no mutation has '
+         'been found.',
+    '3': 'The molecular basis for the disorder is known; a mutation has been found in the gene.',
+    '4': 'A contiguous gene deletion or duplication syndrome, multiple genes are deleted or duplicated causing '
+         'the phenotype.',
+}
+# todo: double check / handle edge cases if/as needed for any concerns:
+#  - https://github.com/monarch-initiative/omim/issues/79#issuecomment-1319408780
+# MORBIDMAP_PHENOTYPE_MAPPING_KEY_PREDICATES provenance:
+# - https://github.com/monarch-initiative/omim/issues/79#issuecomment-1319408780
+MORBIDMAP_PHENOTYPE_MAPPING_KEY_PREDICATES = {
```

Comment:
@matentzn See my comment here, and Sabrina's above it: https://github.com/monarch-initiative/omim/issues/79#issuecomment-1320997956
For the purposes of merging this PR, I'm not sure if I should:
(a) continue just using `RO_0003303` for everything for now, as we were doing before
(b) Use different `RO` predicates based on Sabrina's default for each `morbidmap.txt` mapping key
(c) Use different `RO` predicates based _a number of different parameters_ as Sabrina discusses

Going with (b) for now until hearing otherwise.